src/iris/api.py

```diff
@@ -1387,6 +1416,22 @@ def on_post(self, req, resp, incident_id):
         except KeyError:
             raise HTTPBadRequest('"owner" field required')
 
+        if owner is not None:
+            connection = db.engine.raw_connection()
+            cursor = connection.cursor()
+            cursor.execute(
+                '''SELECT EXISTS(
+                     SELECT 1
+                     FROM `target`
+                     WHERE `target`.`name` = %s
+                     AND `type_id` = (SELECT `id` FROM `target_type` WHERE `name` = 'user'))''',
```

Comment:
Sorry, one more thing I just noticed: this will allow inactive users. Should we add another WHERE checking the `target`.`active` column?

Reply:
bump